src/pages/Videos/VideoContent/RelativeVideos/RelativeVideos.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import { RelatedSection, RelatedBar, RelatedBarTxt, ScrollDiv } from './RelativeVid.styled';
+import VideoCardRelative from '../RelativeVideos/RelativeVideosDetails';
+// import FormatPubDate from '../../utils/formatPublishedDate';
+
+function VideosRelatives({ idVideo }) {
+  const [dataVideos, setVideoData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [videoPlayer, setVideoPlayer] = useState(idVideo);
+
+  useEffect(() => {
+    const fetchData = async () => {

```

Comment:
Once you create the custom hook I mentioned, you can call it from there instead of calling the API here again. You can pass more params to the custom hook (such as the related video id needed here)